main/webservices/registration.soap.php

```diff
@@ -1850,7 +1853,7 @@ function WSEditUser($params)
 
     if ($user_id == 0) {
         return 0;
-    } else {
+    } else if (empty($enable) || !$enable) {
```

Comment:
Any expression that is considered "false" also matches empty():
http://php.net/empty "A variable is considered empty if it does not exist or if its value equals FALSE. "
Hence you don't need the second part here: " || !$enable)"